solutions/community-deploy/cloud/AWS/variable.tf

```diff
@@ -17,43 +17,49 @@
 #
 variable "security_group_cidr" {
   description = "Value of CIDR block to be used for Security Group. This should be your systems public-ip"
-  type        = string
+  type = string
 }
 
 
 variable "os_version" {
   description = "OS version"
-  type        = string
+  type = string
 }
 
 
 variable "region" {
   description = "Region"
-  type        = string
+  type = string
 }
 
 variable "key_name" {
   description = "SSH key name"
-  type        = string
-  default     = "cortx-key"
+  type = string
+  default = "cortx-key"
 }
 
 variable "instance_count" {
   description = "EC2 instance count"
-  type        = number
-  default     = "1"
+  type = number
+  default = "1"
```

Comment:
should it be `3` as default as our document is inclined towards 3 node setup?

Reply:
yes